lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
I really don't like the current tag name though. Can we use a Boolean flag
for :otp19 then? And we exclude based on true and false.

On Thursday, June 23, 2016, Eric Entin <notifications@github.com> wrote:

> In lib/elixir/test/elixir/test_helper.exs
> <https://github.com/elixir-lang/elixir/pull/4855#discussion_r68151089>:
>
> > @@ -1,4 +1,10 @@
> > -ExUnit.start [trace: "--trace" in System.argv]
> > +exclude =
> > +  case :erlang.system_info(:otp_release) do
>
> Ok, so I'm gonna stick with it as is so that the tests start running again
> on OTP 20? Does that sound ok?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4855/files/1bd0b165454dd143d26c34444bdb599faa942b2a#r68151089>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAlbjt2iwzcU19WfKbQVrDCCB6H1CI4ks5qOb-3gaJpZM4I8OnO>
> .
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*